codesearch/schema/schema.go

```diff
@@ -30,10 +31,20 @@ var (
 		MustFieldSchema(types.KeywordField, RepoField, true),
 		MustFieldSchema(types.KeywordField, SHAField, true),
 	})
+	metadataSchema = NewDocumentSchema([]types.FieldSchema{
+		// Repository URL
+		MustFieldSchema(types.KeywordField, IDField, false),
+		// Last indexed commit SHA
+		MustFieldSchema(types.KeywordField, LatestSHAField, true),
+	})
 )
 
-func DefaultSchema() types.DocumentSchema {
-	return defaultSchema
+func GitHubFileSchema() types.DocumentSchema {
```

Comment:
I won't tell you how long I spent thinking about if this was a good name or not. Anyway I got nothing better.